src/forms/reset-password-popup/forgot-password/index.jsx

```diff
@@ -168,7 +171,7 @@ const ForgotPasswordForm = () => {
         <ForgotPasswordSuccess email={formFields.email} />
       )}
       <div className="text-center mt-4.5">
-        {loginErrorCode !== REQUIRE_PASSWORD_CHANGE && (
+        {loginErrorCode !== REQUIRE_PASSWORD_CHANGE && !authenticatedUser && (
```

Comment:
why do we have `!authenticatedUser` check here? hasn't the user been authenticated on nudge?

Reply:
We need to hide the back to login button in case nudge case that is why i put it here

Reply:
i understand the purpose. But i want to understand the `!authenticatedUser` check? Why are we applying it for hiding login button?

Reply:
This is applied as per the ticket instruction in password nudge case we need to hide the button is there any confusion please share with me

Reply:
yes, basically the nudge state is authenticated state of user. so if we are applying !authenticatedUser we are negating the fact that user is authenticated at this point of flow.
this is working because the authenticatedUser just doesn't update itself

please correct my understanding if its wrong